### PR TITLE
Wordpress Example for vmwscript

### PR DESCRIPTION
**Pull request by thebsdbox**
This will take a CentOS template, update it and configure and install
wordpress. Remove the static networking for DHCP.

This is purely a PoC for end-users to learn more about vmwscript.

Signed-off-by: Dan Finneran <daniel.finneran@gmail.com>

### CODE REVIEW

File: pkg/x/vmwscript/examples/wordpress.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                   {
+                    "type":"execute",                    
+                    "note":"Creating directories and set directory permissions",            
+                    "cmd":"mkdir -p /var/www/html/wp-content{uploads,cache}; chown apache:apache /var/www/html/wp-content{uploads,cache}",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Create Wordpress configuration file and add configuration details",            
+                    "cmd":"cp /var/www/html/wp-config-sample.php /var/www/html/wp-config.php; sed -i 's@database_name_here@wordpress@' /var/www/html/wp-config.php; sed -i 's@username_here@wordpress@' /var/www/html/wp-config.php;sed -i 's@password_here@password@' /var/www/html/wp-config.php;",
```
Comment (chungers) [inline pkg/x/vmwscript/examples/wordpress.json]: Use the playbook to turn these into flags / prompts from the user when running the script?

File: pkg/x/vmwscript/examples/wordpress.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                   {
+                    "type":"execute",                    
+                    "note":"Enable DB on Boot and start",            
+                    "cmd":"/usr/bin/systemctl enable mariadb.service; /usr/bin/systemctl start mariadb.service",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Configure Database",            
+                    "cmd":"echo 'CREATE DATABASE wordpress;' | mysql; echo \"GRANT ALL PRIVILEGES ON wordpress.* TO 'wordpress'@'localhost' IDENTIFIED BY 'password';\" | mysql; echo \"FLUSH PRIVILEGES;\" | mysql",
```
Comment (chungers) [inline pkg/x/vmwscript/examples/wordpress.json]: Database password...

File: pkg/x/vmwscript/examples/wordpress.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
+{
+    "label":"Wordpress",
+    "version":"0.1",
+    "vmconfig" : {
+        "guestCredentials" : {
+            "guestUser" : "root",
+            "guestPass" :"password"
```
Comment (chungers) [inline pkg/x/vmwscript/examples/wordpress.json]: Flags...

**Comment by thebsdbox** [comment]
Yeah, I’ll re-write it to be a Wordpress playbook. Or perhaps even a
markdown showing the transition from a scripted install as a case study?

**Pull request merged**